node/Cargo.toml

```diff
@@ -105,6 +102,14 @@ polkadot-cli = { git = "https://github.com/paritytech/polkadot", branch = "rococ
 polkadot-test-service = { git = "https://github.com/paritytech/polkadot", branch = "rococo-v1" }
 polkadot-parachain = { git = "https://github.com/paritytech/polkadot", branch = "rococo-v1" }
 
+# benchmarking dependencies
+frame-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "rococo-v1" }
+frame-benchmarking-cli = { git = "https://github.com/paritytech/substrate", branch = "rococo-v1" }
+pallet-xcm = { git = "https://github.com/paritytech/polkadot", branch = "rococo-v1", optional = true }
+xcm-builder = { git = "https://github.com/paritytech/polkadot", branch = "rococo-v1", optional = true }
+pallet-collective = { git = "https://github.com/paritytech/substrate", branch = "rococo-v1", optional = true }
+pallet-society = { git = "https://github.com/paritytech/substrate", branch = "rococo-v1", optional = true }
```

Comment:
These seem irrelevant. Especially pallet-society.

Reply:
I actually agree with this statement, but I get a compile error without `pallet-society`:

```
error[E0046]: not all trait items implemented, missing: `successful_origin`
    --> /home/stephen/.cargo/git/checkouts/substrate-7e08433d4c370a21/68cfc32/frame/society/src/lib.rs:1163:1
     |
1163 | impl<T: Config> EnsureOrigin<T::Origin> for EnsureFounder<T> {
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `successful_origin` in implementation
     |
     = help: implement the missing item: `fn successful_origin() -> OuterOrigin { todo!() }`
```

I think it's worth understanding this dependency if it' required / removing it if not.

Reply:
@shawntabrizi Do you know anything about this? pallet-society is in our dependency graph only because of `polkadot-service`.

```
joshy@scg-moonbuild-1:~/moonbeam$ cargo tree -i pallet-society
pallet-society v3.0.0 (https://github.com/paritytech/substrate?branch=rococo-v1#68cfc326)
├── kusama-runtime v0.8.30 (https://github.com/paritytech/polkadot?branch=rococo-v1#943038a8)
│   └── polkadot-service v0.8.30 (https://github.com/paritytech/polkadot?branch=rococo-v1#943038a8)
│       ├── cumulus-client-consensus-relay-chain v0.1.0 (https://github.com/paritytech/cumulus?branch=rococo-v1#6ed96321)
│       │   └── moonbeam v0.7.0 (/home/joshy/moonbeam/node)
│       ├── cumulus-client-network v0.1.0 (https://github.com/paritytech/cumulus?branch=rococo-v1#6ed96321)
│       │   ├── cumulus-client-collator v0.1.0 (https://github.com/paritytech/cumulus?branch=rococo-v1#6ed96321)
│       │   │   ├── cumulus-client-service v0.1.0 (https://github.com/paritytech/cumulus?branch=rococo-v1#6ed96321)
│       │   │   │   └── moonbeam v0.7.0 (/home/joshy/moonbeam/node)
│       │   │   └── moonbeam v0.7.0 (/home/joshy/moonbeam/node)
│       │   └── moonbeam v0.7.0 (/home/joshy/moonbeam/node)
│       ├── cumulus-client-service v0.1.0 (https://github.com/paritytech/cumulus?branch=rococo-v1#6ed96321) (*)
│       ├── moonbeam v0.7.0 (/home/joshy/moonbeam/node)
│       ├── polkadot-cli v0.8.30 (https://github.com/paritytech/polkadot?branch=rococo-v1#943038a8)
│       │   └── moonbeam v0.7.0 (/home/joshy/moonbeam/node)
│       └── polkadot-test-service v0.8.30 (https://github.com/paritytech/polkadot?branch=rococo-v1#943038a8)
│           └── moonbeam v0.7.0 (/home/joshy/moonbeam/node)
│           [dev-dependencies]
│           └── moonbeam v0.7.0 (/home/joshy/moonbeam/node)
└── westend-runtime v0.8.30 (https://github.com/paritytech/polkadot?branch=rococo-v1#943038a8)
    └── polkadot-service v0.8.30 (https://github.com/paritytech/polkadot?branch=rococo-v1#943038a8) (*)
```

Reply:
I was able to remove `pallet-collective` in 5a4cc53 but I get compile errors if I touch any of the other 3.